tests/lib/northbound/test_oper_exists.c

```diff
@@ -0,0 +1,265 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/*
+ * Copyright (C) 2018  NetDEF, Inc.
+ *                     Renato Westphal
```

Comment:
Renato?  Did you do this Chris or did Renato?

Reply:
Well I copied the test_oper_data.c, the copyright came with it. :)

Reply:
appended copyright